src/khepri.erl

```diff
@@ -2594,8 +2597,162 @@ wait_for_async_ret(Correlation, Timeout) ->
+%%
+%% Example: export the full Khepri store using {@link khepri_export_erlang} as
+%% the callback module
+%% ```
+%% ok = khepri:export(StoreId, khepri_export_erlang, "export-1.erl").
+%% '''
+%%
+%% Example: export a subset of the Khepri store
+%% ```
+%% ok = khepri:export(
+%%        StoreId,
+%%        "/:stock/:wood/**",
+%%        khepri_export_erlang,
+%%        "export-wood-stock-1.erl").
+%% '''
+%%
+%% @param StoreId the name of the Khepri store.
+%% @param PathPattern the path pattern matching the tree nodes to export.
+%% @param Module the callback module to use to export.
+%% @param ModulePriv arguments passed to `Module:open_write/1'.
+%%
+%% @returns `ok' or an `{ok, Term}' tuple if the export succeeded (the actual
+%% return value depends on whether the callback module wants to return anything
+%% to the caller), or an `{error, Reason}' tuple if it failed.
+%%
+%% @see import/3.
+
+export(StoreId, PathPattern, Module, ModulePriv)
+  when ?IS_STORE_ID(StoreId) andalso is_atom(Module) ->
+    khepri_import_export:export(StoreId, PathPattern, Module, ModulePriv).
+
+-spec import(Module, ModulePriv) -> Ret when
+      Module :: module(),
+      ModulePriv :: khepri_import_export:module_priv(),
+      Ret :: ok | {ok, ModulePriv} | {error, any()}.
+%% @doc Imports a previously exported set of tree nodes using the `Module'
+%% callback module.
+%%
+%% Calling this function is the same as calling `import(StoreId, Module,
+%% ModulePriv)' with the default store ID (see {@link
+%% khepri_cluster:get_default_store_id/0}).
+%%
+%% @see import/3.
+
+import(Module, ModulePriv) when is_atom(Module) ->
+    StoreId = khepri_cluster:get_default_store_id(),
+    import(StoreId, Module, ModulePriv).
+
+-spec import(StoreId, Module, ModulePriv) -> Ret when
+      StoreId :: khepri:store_id(),
+      Module :: module(),
+      ModulePriv :: khepri_import_export:module_priv(),
+      Ret :: ok | {ok, ModulePriv} | {error, any()}.
+%% @doc Imports a previously exported set of tree nodes using the `Module'
+%% callback module.
+%%
+%% `Module' is the callback module called to perform the actual import. It
+%% must conform to the Mnesia Backup &amp; Restore API. See {@link
+%% khepri_import_export} for more details.
+%%
+%% `ModulePriv' is the term passed to `Module:open_read/1'.
+%%
+%% Importing something doesn't delete existing tree nodes. The caller is
+%% responsible for deleting the existing content of a store if he needs to.
+%%
+%% Example: import a set of tree nodes using {@link khepri_export_erlang} as
+%% the callback module
+%% ```
+%% ok = khepri:import(StoreId, khepri_export_erlang, "export-1.erl").
+%% '''
+%%
+%% @param StoreId the name of the Khepri store.
+%% @param Module the callback module to use to import.
+%% @param ModulePriv arguments passed to `Module:open_read/1'.
+%%
+%% @returns `ok' or an `{ok, Term}' tuple if the import succeeded (the actual
+%% return value depends on wheter the callback module wants to return anything
```

Comment:
```suggestion
%% return value depends on whether the callback module wants to return anything
```

Reply:
Thanks, I missed the fact you mentionned the mistake was in two places.